projects/storefrontlib/src/cms-components/asm/asm-enabler.service.ts

```diff
@@ -0,0 +1,94 @@
+import { Location } from '@angular/common';
```

Comment:
Minor points:
- AsmEnablerService could be moved to the 'services' folder.
- Not saying we should, but would there be benefits to add AsmEnablerService functions to AsmComponentService instead?  
- I know this is a work in progress, but so we don't forget: the service needs unit tests.

Reply:
Reminder to remove the visible attribute from the AsmUi model.

Reply:
- moved
- I prefer to keep the `AsmEnablerService` and potentially refactor the `AsmComponentService` to take on more responsibilities from the components (but not enabling) 
- test have been added
- visibility flag dropped